apiserver/facades/client/bundle/bundle.go

```diff
@@ -430,6 +432,13 @@ func (b *BundleAPI) fillBundleData(model description.Model) (*bundleOutput, erro
 
 		data.Machines[machine.Id()] = newMachine
 	}
+
+	for _, application := range model.RemoteApplications() {
```

Comment:
You will need (for now) a `if featureflag.Enabled(feature.CMRAwareBundles)` guard here